src/types/Client.d.ts

```diff
@@ -37,7 +37,7 @@ export interface QueryOptions
   > {
   signal?: AbortSignal
   traceparent?: string
-  tags?: { string: string }
+  tags?: { [key: string]: string }
```

Comment:
I'm a little confused; when I was running queries with multiple tags, they were showing up correctly in the logs.  Shouldn't I expect that not to work?

Reply:
did you use typescript or raw javascript?

If you use types it would refuse to compile.

Reply:
Oh right, that's the ticket.  Thanks.

Reply:
> Oh, that makes sense. We'll need to bump that to `-beta.2` before releasing also.

Sure I can do that now.